compiler/src/linking/link.re

```diff
@@ -135,7 +135,16 @@ let is_function_imported = func =>
 //       constructing the AST (either through constructing two separate instances or by
 //       using Expression.copy())
 
-let rec globalize_names = (local_names, expr) => {
+let rec globalize_names = (function_names, global_names, label_names, expr) => {
```

Comment:
Any reason that `function_names`, `global_names` and `label_names` aren't named parameters?